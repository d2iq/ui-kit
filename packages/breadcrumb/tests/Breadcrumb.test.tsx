import React from "react";
import { render } from "@testing-library/react";

import { Breadcrumb } from "../";

describe("Breadcrumb", () => {
  it("default", () => {
    const { asFragment } = render(
      <Breadcrumb>
        <span>One</span>
        <span>Two</span>
      </Breadcrumb>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
