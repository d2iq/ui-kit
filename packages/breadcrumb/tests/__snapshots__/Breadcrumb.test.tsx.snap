// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb default 1`] = `
<nav
  class="css-cldwuj"
>
  <div
    class="css-zsjup2"
    data-cy="breadcrumb"
  >
    <span>
      One
    </span>
  </div>
  <div
    class="css-10fubqe"
    data-cy="breadcrumb"
  >
    <svg
      aria-label="system-caret-right icon"
      class="css-jbdp75"
      data-cy="icon"
      height="16"
      preserveAspectRatio="xMinYMin meet"
      role="img"
      viewBox="0 0 16 16"
      width="16"
    >
      <use
        href="#system-caret-right"
      />
    </svg>
  </div>
  <div
    class="css-10fubqe"
    data-cy="breadcrumb"
  >
    <span>
      Two
    </span>
  </div>
</nav>
`;
